MementoMori/mementomori.client/src/decks/Deck.tsx

```diff
@@ -90,8 +103,11 @@ function Buttons(props: ButtonProps) {
     window.location.href = `/decks/${deckId}/practice`;
   };
   const onAddToMyCollectionClick = () => {
-    // send request to backend to verify that can add and then add
+    // send request to backend, verify that can add and then add
+    // send request to backend, verify that can add and then add
     // show spinner until response
+    AddToCollection();
```

Comment:
AddToCollection(); should not be repeated twice. Seems to be a left over from dealing with merge conflicts.